CHANGELOG.md

```diff
@@ -2,15 +2,137 @@
 
 ## Unreleased
 
+## v0.107.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.107.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.107.0) and the [opentelemetry-collector-contrib v0.107.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.107.0) releases where appropriate.
+
+This release fixes CVE-2024-42368 on the `bearerauthtokenextension` ([#34516](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34516)). The Splunk distribution was not impacted by this CVE.
+
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) `confmap`: Do not expand special shell variable such as `$*` in configuration files. ([#5206](https://github.com/signalfx/splunk-otel-collector/pull/5206))
+- (Core) `service`: Remove OpenCensus bridge completely, mark feature gate as stable. ([#10414](https://github.com/open-telemetry/opentelemetry-collector/pull/10414))
+- (Core) `confmap`: Set the `confmap.unifyEnvVarExpansion` feature gate to Stable. Expansion of `$FOO` env vars is no longer supported.  Use `${FOO}` or `${env:FOO}` instead. ([#10508](https://github.com/open-telemetry/opentelemetry-collector/pull/10508))
+
```

Comment:
nit of neatness: there is no blank line from Splunk to Core items, I do not have preference, but it should be consistent

Reply:
Thanks for pointing this out, agreed we should be consistent. I've added newlines between different distributions within each section.